app/controllers/discussions_controller.rb

```diff
@@ -1,8 +1,9 @@
 class DiscussionsController < ApplicationController
 
-  before_action :set_event, only: [:index, :show, :new, :edit, :create, :update]
+  before_action :set_event, except: [:destroy]
```

Comment:
Should I change it?

On Fri, Jul 31, 2015 at 4:21 PM, Jaspreet Singh <notifications@github.com>
wrote:

> In app/controllers/discussions_controller.rb
> <https://github.com/SAkshat/iconf/pull/3#discussion_r35963947>:
>
> > @@ -1,8 +1,9 @@
> >  class DiscussionsController < ApplicationController
> >
> > -  before_action :set_event, only: [:index, :show, :new, :edit, :create, :update]
> > +  before_action :set_event, except: [:destroy]
>
> Okay, but its better to use only since it would prevent unintentional
> calling of method if you add any other action to controller
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SAkshat/iconf/pull/3/files#r35963947>.
>



-- 

Akshat Singhal
Vinsol